bin/rwhois

```diff
@@ -48,9 +48,7 @@ if ARGV.empty?
 end
 
 begin
-  ips = Socket.ip_address_list.select{|intf| intf.ipv4? and !intf.ipv4_loopback? and !intf.ipv4_multicast?}
```

Comment:
This is required to rotate through external IP addresses. Otherwise, all requests come from the same IP address, and the whois servers block.

Reply:
Ah, didn't know it was on purpose. Added back.